brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1010 Fix: Responsive Range Control - Unset issue.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1010

### Screenshots

- Fix - https://d.pr/v/yzc0ud

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop any block. In this case, I used the Heading block.
- Enable the Separator and go to Style > Separator.
- Set width to 0 and click outside of the option it should be set 0 same as Thinkness range control.

### Checklist:
